src/utils/Utils.js

```diff
@@ -117,43 +117,6 @@ export function validateDocumentName(name = "") {
   }
 }
 
-const boolToPythonOptions = {
-  true: "True",
-  false: "False"
-};
-
-const PythonToBoolOptions = {
-  True: true,
```

Comment:
You could also use these consts to help with validations in the boolean datatype.

Reply:
No longer needed. We have parse and uparse for that